src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -114,7 +127,14 @@ public void focus() {
     @FXML
     private void openUrl() {
         try {
-            Desktop.getDesktop().browse(new URI(USERGUIDE_URL));
+            if (System.getProperty("os.name").trim().toLowerCase().contains(OS.LINUX.toString().toLowerCase())) {
```

Comment:
would this benefit from using the `AppUtil.isLinux()` method?

Reply:
Good catch! Already made the changes